### PR TITLE
lwm2m 2024 release note updates

### PR DESCRIPTION
**Pull request by Oezge-Akin**

### CODE REVIEW

File: content/change-logs/device-management/cumulocity-10-18-540-11-lwm2m-disable-default-object-actions.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+  - value: component-1KLUzmqfe
+    label: LWM2M
+build_artifact:
+  - value: tc-QHwMfWtBk7
+    label: cumulocity
+ticket: DM-1849
+version: 10.18.540.11
+---
+
+The default handling of object instance actions for LWM2M objects 3, 4 and 6 can now be turned off. In addition, the default server-side firmware update state machine for object 5 can be disabled. These settings can be enabled/disabled in the device´s LWM2M  configuration in the UI.
```
Comment (BeateRixen) [inline content/change-logs/device-management/cumulocity-10-18-540-11-lwm2m-disable-default-object-actions.md:18]:
```suggestion
The default handling of object instance actions for LWM2M objects 3, 4 and 6 can now be turned off. In addition, the default server-side firmware update state machine for object 5 can be disabled. These settings can be enabled/disabled in the device´s LWM2M configuration in the UI.
```

File: content/change-logs/device-management/cumulocity-10-18-540-36-lwm2m-handling-location-object-with-too-high-precision.md [hunk 2]
```diff
@@ -0,0 +1,18 @@
+---
+date: 2024-03-28
+title: LWM2M agent location object 6 with high precision
```
Comment (BeateRixen) [inline content/change-logs/device-management/cumulocity-10-18-540-36-lwm2m-handling-location-object-with-too-high-precision.md:3]:
```suggestion
title: LWM2M agent correctly rounds values from location object 6
```

Comment (BeateRixen) [reply content/change-logs/device-management/cumulocity-10-18-540-36-lwm2m-handling-location-object-with-too-high-precision.md]: @Oezge-Akin Please check. The title should always state what works, not was is broken. The current title actually does not say anything. It often better to have a verb in the title.

File: content/change-logs/device-management/cumulocity-10-18-540-56-renaming-endpoint-name-with-LWM2M-configuration-ui.md [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+  - value: component-1KLUzmqfe
+    label: LWM2M
+build_artifact:
+  - value: tc-QHwMfWtBk7
+    label: cumulocity
+ticket: DM-3084
+version: 10.18.540.74
+---
+
+The modification of the LWM2M endpoint name via the device's LWM2M Configuration view now triggers the automatic adjustment of the corresponding external identifier, designated with the type c8y_Id, ensuring seamless synchronization between the endpoint name and its associated external ID.
```
Comment (BeateRixen) [inline content/change-logs/device-management/cumulocity-10-18-540-56-renaming-endpoint-name-with-LWM2M-configuration-ui.md:18]:
```suggestion
The modification of the LWM2M endpoint name via the device's LWM2M configuration now triggers the automatic adjustment of the corresponding external identifier, designated with the type c8y_Id, ensuring seamless synchronization between the endpoint name and its associated external ID.
```

File: content/change-logs/device-management/cumulocity-10-18-540-56-renaming-endpoint-name-with-LWM2M-configuration-ui.md [hunk 4]
```diff
@@ -0,0 +1,18 @@
...
+  - value: component-1KLUzmqfe
+    label: LWM2M
+build_artifact:
+  - value: tc-QHwMfWtBk7
+    label: cumulocity
+ticket: DM-3084
+version: 10.18.540.74
+---
+
+The modification of the LWM2M endpoint name via the device's LWM2M configuration now triggers the automatic adjustment of the corresponding external identifier, designated with the type c8y_Id, ensuring seamless synchronization between the endpoint name and its associated external ID.
```
Comment (BeateRixen) [inline content/change-logs/device-management/cumulocity-10-18-540-56-renaming-endpoint-name-with-LWM2M-configuration-ui.md:18]:
```suggestion
The modification of the LWM2M endpoint name via the device's LWM2M configuration view now triggers the automatic adjustment of the corresponding external identifier, designated with the type c8y_Id, ensuring seamless synchronization between the endpoint name and its associated external ID.
```

File: content/change-logs/device-management/cumulocity-10-18-540-56-renaming-endpoint-name-with-LWM2M-configuration-ui.md [hunk 5]
```diff
@@ -0,0 +1,18 @@
+---
+date: 2024-03-28
+title: LWM2M agent changing endpoint name using LWM2M Configuration UI
```
Comment (BeateRixen) [inline content/change-logs/device-management/cumulocity-10-18-540-56-renaming-endpoint-name-with-LWM2M-configuration-ui.md:3]:
```suggestion
title: Changing the endpoint name in the LWM2M configuration view automatically adjusts external ID
```

**Review by jakubsobolewskisag — APPROVED** [review]

File: content/change-logs/device-management/cumulocity-10-18-540-11-lwm2m-disable-default-object-actions.md [hunk 6]
```diff
@@ -0,0 +1,18 @@
+---
+date: 2023-03-28
+title: Disabling default object actions using LWM2M configuration tab
```
Comment (BeateRixen) [inline content/change-logs/device-management/cumulocity-10-18-540-11-lwm2m-disable-default-object-actions.md:3]:
```suggestion
title: Disabling default object actions in the LWM2M configuration UI
```

**Review by BeateRixen — APPROVED** [review]

**Pull request merged**